src/courses/mappers/03.md

```diff
@@ -7,17 +7,26 @@ author: Charles Hu
 
 ## Where Is OHDF Used?
 
-The SAF has five core capabilities, which you can read more about [here](../user/03.md). Here is how OHDF relates to these five pillars:
+OHDF is a cornerstone of the Security Automation Framework and is officially implemented and used in a plethora of tools and libraries including InspecJS, OHDF Converters, the SAF CLI, and Heimdall.
 
-- **Plan**: The OHDF spec is written such that it includes all the information described in the guidance, as well as all the relavant metadata. Thus, the information is carried alongside the results. 
-- **Harden**: The OHDF spec is not really relevant in this case. 
-- **Validate**: InSpec's results format is a subset of the OHDF spec. This means that InSpec itself is natively supported! Moreover, it's easy to write InSpec such that all the guidance information is passed through into both the OHDF profile and OHDF results formats. 
-- **Normalize**: Normalization is the primary use of OHDF. With the use of OHDF Converters, we are able to map many different security results formats to a standard format (OHDF). These converters can either be implemented directly, or accessed via the SAF CLI. 
-- **Visualize**: OHDF is ingested by a variety of tools (ex. Heimdall) and transformed in many different ways. For example, a results file can be turned into a threshold file for pipeline use, turned into a POAM for ingestion by eMASS, or ingested by Heimdall directly for display purposes. 
+## What Is the Security Automation Framework?
 
+The Security Automation Framework (SAF) is a DevSecOps framework which provides a suite of tools for assessing, applying, and managing the security compliance of computer systems. SAF has five core capabilities, of which OHDF is involved in varying degrees:
+
+- **Plan**: OHDF is defined such that it includes all information described in the guidance, as well as relavant meta data. Thus, the information provided by the guidance are incorporated with the security tool results.
```

Comment:
"The OHDF schema is defined such that it can include all information..."

relevant spelled wrong, metadata should be one word, information is a singular noun so use 'is' not 'are'

Reply:
Addressed.